### ISSUE TITLE
Input error

### ISSUE THREAD

**Issue by Gorgho** [flaviostutz/ruller#5]
Now if i send a request without input, i will reicive an error.
How about create a default where if condition ask for input and i don't send that, the app continue just not aplly that condition?

**Comment by flaviostutz** [comment]
What do you think, @tiagostutz @jairsjunior @abilioesteves?

@Gorgho could you please send a real example on this?

**Comment by flaviostutz** [comment]
@rafamarts

**Comment by abilioesteves** [comment]
@flaviostutz, I think this is a Domain Specific Issue, meaning only the client app would know the behaviour it wants to see. 

Although it's a weak link, we do have a semantic relationship between `client apps` and `rule groups` as heavily suggested by `feature-dsl`'s examples.

Based on that, we could add to each rule group a boolean `lazyEvaluation` field that tells the Ruller the right evaluation behaviour for a specific group: 

- `lazyEvaluation == true`: if a required input is not provided, return error;
- `lazyEvaluation == false`: if a required input is not provided, evaluates all the conditions that does not makes use of such input;

What do you think?

**Comment by jairsjunior** [comment]
I think that is a good option has the property `lazyEvaluation` because with this the evaluation can be not so strict if the client app permits.

**Comment by flaviostutz** [comment]
Indeed, I think this would be a DSL option, not a feature for the core ruller. It would be a "config" option on the json itself or something. Lets continue this discussion here: https://github.com/flaviostutz/ruller-dsl-feature-flag/issues/11

**Comment by abilioesteves** [comment]
There is a transfer issue option on github: 

![image](https://user-images.githubusercontent.com/6494839/83923415-9828e900-a758-11ea-9402-113108c854dc.png)
